### DISCUSSION TITLE
Using Smarthost or relay

### DISCUSSION THREAD

**Discussion by bugraceylan** [postalserver/postal#1893] [Help with using Postal]
Hello developer(s),

Thank you so much for your amazing work. I wonder if its possible to route outbound emails to another smtp server like o365 or amazon SES? Something like this:

Client ----> Postal ----> Amazon SES ----> Destination

**Comment by willpower232** [comment]
Postal can make use of relays for outgoing email https://github.com/postalserver/postal/blob/d4aebac3cb398a193cd8c753d2f3a17cd3339ce9/config/postal.defaults.yml#L74-L78

However these cannot have authentication on them so I'm not sure if you'll be able to do that because SES probably requires authentication.

**Reply by bugraceylan** [reply]
Thanks @willpower232, you are right without authentication, It won't be usable for me. I hope this functionality can be added in the future.
